lib/cdo/shared_constants.rb

```diff
@@ -52,6 +52,8 @@ module SharedConstants
 
   STUDENT_GRADE_LEVELS = %w(K 1 2 3 4 5 6 7 8 9 10 11 12 Other).freeze
 
+  PL_GRADE_VALUE = 'pl'.freeze
```

Comment:
special values are concerning. can you help me understand why this is necessary, rather than using other properties of the section to determine whether it is a PL section?

Reply:
after offline conversation, it sounds like we will not have logic predicated on this field and it is just needed for RED purposes. I think this would be great to capture in a comment

Reply:
also, having a validation here eliminates my worry about duplicate state getting out of sync 👍